src/app/Login/page.tsx

```diff
@@ -0,0 +1,97 @@
+'use client';
+import React, { useState } from 'react';
+import ActiveButton from '@/components/Active-Button/ActiveButton';
+import InactiveButton from '@/components/Inactive-Button/InactiveButton';
+import InputField from '@/components/Input-textbox/textbox';
+import styles from './page.module.css';
+import { emailRegex } from '@/utils/constants';
+
+const isValidEmail = (email: string) => {
+  return emailRegex.test(email);
+};
+
+const LoginPage: React.FC = () => {
+  // State for email and password
+
+  const [email, setEmail] = useState('');
+  const [password, setPassword] = useState('');
+  const [emailError, setEmailError] = useState<string | null>(null);
+
+  // Handlers for input changes
+  const handleEmailChange = (e: React.ChangeEvent<HTMLInputElement>) => {
+    const value = e.target.value;
+    setEmail(value);
+
+    // Validate email on change
+    if (!isValidEmail(value)) {
+      setEmailError('Please enter a valid email address.');
+    } else {
+      setEmailError(null);
+    }
```

Comment:
instead of validating email on change, it is standard to validate the email on submit. We want to minimize error messages to the user and only display them if for certain they have entered invalid input, not while they are entering a possibly valid email in this case.

Reply:
you already are validating the email on submit so you can just delete this chunk of code